google/cloud/bigquery/client.py

```diff
@@ -311,6 +311,10 @@ def list_projects(
                 The number of seconds to wait for the underlying HTTP transport
                 before using ``retry``.
 
+            page_size (Optional[int]):
+                Maximum number of projects to return in each page, If not passed,
```

Comment:
(nit) Seems like a fullstop was meant here instead of a comma. :)
```suggestion
                Maximum number of projects to return in each page. If not passed,
```

Reply:
No, although arguably, "If not passed, " could be omitted. This language was copied from `max_results` above. I'll change the wording because fewer words is better. :)

Reply:
done